R/FilterState.R

```diff
@@ -313,8 +315,8 @@ FilterState <- R6::R6Class( # nolint
         )
       )
       value <- private$cast_and_validate(value)
-      value <- private$remove_out_of_bound_values(value)
-      private$validate_selection(value)
+      # value <- private$remove_out_of_bound_values(value)
+      # private$validate_selection(value)
```

Comment:
remove :) - also are we OK removing this? Is it a problem for other classes as it lives in the base class?

Reply:
This was actually moved to the `format` method of `FilterStateRanged`, so other `format` methods should (and will) be modified accordingly.